dask_match/datasets.py

```diff
@@ -12,22 +14,43 @@
 
 
 class Timeseries(BlockwiseIO):
-    _parameters = ["start", "end", "dtypes", "freq", "partition_freq", "seed", "kwargs"]
+    _parameters = [
+        "start",
+        "end",
+        "dtypes",
+        "_projection",
```

Comment:
This seems wrong to me.  I would hope to just change `dtypes` to reflect the new columns.

I think that the way I would normally solve this would be to create a separate set of seeds for each column.  Then when we drop a column all of the seeds for the other columns stay the same.  This means more seeds, but hoepfully those are cheap.

Reply:
Okay - Updated the PR to use a random state of length `num-partitions * num-columns`. Sorry, I overlooked the fact that we have control of `make_timeseries_part`. My mistake.